department-of-veterans-affairs/caseflow
# Replace Vocational Rehab benefit type

Resolves [CASEFLOW-2621](https://vajira.max.gov/browse/CASEFLOW-2621)
Related to #16918
### Description
Resolve showing "Vocational Rehabilitation and Employment" benefit type after intake

### Acceptance Criteria
- [ ] "Vocational Rehabilitation and Employment" does not show on business line queue page
- [ ] "Vocational Rehabilitation and Employment" does not show within the table

### Testing Plan

1. Sign in as intake user
2. Begin intaking supplemental claim
3. Select "Veterans Readiness and Employment" benefit type
4. Fill out the rest of the form
5. Continue to add issues page
6. Add any issue, and click "Establish Claim"
7. Once the claim is established, the user is directed to the Business Line Queue page.  
8. Click on the claimant name to be directed to the Decision Review page.  Here the old benefit type language shows again at the top of the page and table header.  See screenshot attached (Bug_SC_2)

### User Facing Changes

 BEFORE|AFTER
 ---|---
TBA
